lib/init-option-defaults.js

```diff
@@ -11,5 +18,17 @@ module.exports = function (bundle) {
   bundle[BundleKeys.OPTIONS].pluginOptions['gulp-minify-css'] = bundle[BundleKeys.OPTIONS].pluginOptions['gulp-minify-css'] || {};
   bundle[BundleKeys.OPTIONS].pluginOptions['gulp-uglify'] = bundle[BundleKeys.OPTIONS].pluginOptions['gulp-uglify'] || {};
   bundle[BundleKeys.OPTIONS].pluginOptions['gulp-concat'] = bundle[BundleKeys.OPTIONS].pluginOptions['gulp-concat'] || {};
+
+  bundle[BundleKeys.OPTIONS].pluginOptions['gulp-sourcemaps'] = defaults(bundle[BundleKeys.OPTIONS].pluginOptions['gulp-sourcemaps'] || {}, sourcemapDefaults);
+
+  // This is to get a clone of the base options
+  var sourcemapBaseOpts = defaults({}, bundle[BundleKeys.OPTIONS].pluginOptions['gulp-sourcemaps']);
+  // Remove the style and script sub attributes
```

Comment:
why is this necessary? `sourcemapDefaults` should never be modified (and no scripts/styles key exists in the obj above)?